### ISSUE TITLE
is it working?

### ISSUE THREAD

**Issue by bbday** [DS-GW2/GW2Miner.Engine#1]
I'm looking into your project, i try to understand where you get cookie:

but in my system there aren't any file in location
string.Format("{0}\GW2TP\GW2TP.Config", Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData));

I tried also started gw2 and open black lion but file not exist.

**Comment by DarkSpirit444** [comment]
yes it should be but those APIs that have stopped working have been commented out.
